src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -464,7 +464,7 @@ protected Federation getSpendingFederation(BtcTransaction btcTx) {
         Script redeemScript = extractStandardRedeemScript(getRedeemScriptFromInput(firstInput));
 
         List<Federation> spendingFedFilter = observedFederations.stream()
-                .filter(f -> (f instanceof ErpFederation ? ((ErpFederation) f).getStandardRedeemScript() : f.getRedeemScript()).equals(redeemScript)).collect(Collectors.toList());
+                .filter(f -> (f instanceof ErpFederation ? ((ErpFederation) f).getDefaultRedeemScript() : f.getRedeemScript()).equals(redeemScript)).collect(Collectors.toList());
```

Comment:
If you agree with the above proposal, you can use the method here as well.

Reply:
same here :)